tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -1,15 +1,110 @@
 import pytest
 
-from mlflow.entities import Experiment
+from mlflow.entities import (Experiment, Run, RunInfo, RunData,
+                             Metric, Param, RunTag, ViewType, LifecycleStage)
 
 from mlflow_elasticsearchstore.elasticsearch_store import ElasticsearchStore
 
-actual_experiment = Experiment(experiment_id="WrAh43MB26mjXDfbtLks", name="a.vivien-exp",
-                               lifecycle_stage="active", artifact_location="artifact_path", tags=[])
+
+actual_experiment0 = Experiment(experiment_id="hTb553MBNoOYfhXjnnQh", name="exp0",
+                                lifecycle_stage="active", artifact_location="artifact_path",
+                                tags=[])
+
+actual_experiment1 = Experiment(experiment_id="hjb553MBNoOYfhXjp3Tn", name="exp1",
+                                lifecycle_stage="active", artifact_location="artifact_path",
+                                tags=[])
+
+actual_experiment2 = Experiment(experiment_id="hzb553MBNoOYfhXjsXRa", name="exp2",
+                                lifecycle_stage="active", artifact_location="artifact_path",
+                                tags=[])
+
+actual_run_info = RunInfo(run_uuid="7b2e71956f3d4c08b042624a8d83700d",
+                          experiment_id="hTb553MBNoOYfhXjnnQh",
+                          user_id="1",
+                          status="RUNNING",
+                          start_time=1597324762662,
+                          end_time=None,
+                          lifecycle_stage="active",
+                          artifact_uri="artifact_path/7b2e71956f3d4c08b042624a8d83700d/artifacts",
+                          run_id="7b2e71956f3d4c08b042624a8d83700d")
+
+actual_metrics = [Metric(key="metric0", value=15.0, timestamp=1597324762700, step=0),
+                  Metric(key="metric0", value=7.0, timestamp=1597324762742, step=1),
+                  Metric(key="metric0", value=20.0, timestamp=1597324762778, step=2),
+                  Metric(key="metric1", value=20.0, timestamp=1597324762815, step=0),
+                  Metric(key="metric1", value=0.0, timestamp=1597324762847, step=1),
+                  Metric(key="metric1", value=7.0, timestamp=1597324762890, step=2)]
+
+actual_params = [Param(key="param0", value="val2"),
+                 Param(key="param1", value="Val1"),
+                 Param(key="param2", value="Val1"),
+                 Param(key="param3", value="valeur4")]
+
+actual_tags = [RunTag(key="tag0", value="val2"),
+               RunTag(key="tag1", value="test3"),
+               RunTag(key="tag2", value="val2"),
+               RunTag(key="tag3", value="test3")]
+
+actual_run_data = RunData(metrics=actual_metrics, params=actual_params, tags=actual_tags)
+
+actual_run = Run(run_info=actual_run_info, run_data=actual_run_data)
 
 
 @pytest.mark.integration
```

Comment:
Instead of using this before each test, you can just put `pytestmark = pytest.mark.integration` at the beginning of the file